tests/app/main/test_frameworks.py

```diff
@@ -3608,17 +3572,15 @@ def test_names_with_the_section_name_in_them_will_display_correctly(self, s3):
         doc = html.fromstring(response.get_data(as_text=True))
         self._assert_page_title_and_table_headings(doc)
 
-        tables = doc.xpath('//div[contains(@class, "updates-document-tables")]/table')
+        [communications_table, clarifications_table] = doc.xpath('//table[contains(@class, "govuk-table")]')
```

Comment:
I guess this may be a bit brittle as it will find the first table on the page, so maybe better to add a `data-name` or similar specific attribute that would be unique to this table

Reply:
I made the xpath more specific, so it should now be less brittle.

Reply:
Actually, looking at it some more, I think a `data-name` is more elegant